apps/apps.c

```diff
@@ -649,10 +653,12 @@ X509 *load_cert(const char *file, int format, const char *cert_descrip)
         x = d2i_X509_bio(cert, NULL);
     } else if (format == FORMAT_PEM) {
         x = PEM_read_bio_X509_AUX(cert, NULL,
-                                  (pem_password_cb *)password_callback, NULL);
+                                  (pem_password_cb *)password_callback, &cb_data);
```

Comment:
I also see little point doing so for a cert-only file, but 
* a PEM file might contain also other structures including private keys, and it looks like in such case the while file can be encrypted (similarly to PKCS#12 files) and
* `PEM_read_bio_X509_AUX()` does support a password callback argument, 

so at least it should not hurt supplying it just in case.